backend/.env

```diff
@@ -0,0 +1,13 @@
+GRYNOS_COURSES_UPDATE_INTERVAL=3600000
```

Comment:
isnt this file supposed go be .env.template

Reply:
it was discussed that .template file can be used in local to keep the original values so that that can be used to overwrite the .env file when required. and the .env file will be committed with just the keys and not values so that each user can fill them.